test/system/irohad_test.cpp

```diff
@@ -154,9 +170,114 @@ class IrohadTest : public AcceptanceFixture {
 
   std::string setDefaultParams() {
     return params(
-        config_copy_, path_genesis_.string(), path_keypair_.string(), {});
+        config_copy_, path_genesis_.string(), path_keypair_node_.string(), {});
   }
 
+  void prepareTestData() {
+    keys_manager_admin_.createKeys();
```

Comment:
I would wrap these calls in `ASSERT_TRUE`.
```suggestion
    ASSERT_TRUE(keys_manager_admin_.createKeys());
```